plenum/server/consensus/checkpoint_service.py

```diff
@@ -287,12 +284,20 @@ def _unknown_stabilized_checkpoints(self) -> List[CheckpointKey]:
     def _is_below_3pc_key(cp: CheckpointKey, key: Tuple[int, int]) -> bool:
         return compare_3PC_keys((cp.view_no, cp.pp_seq_no), key) >= 0
 
+    @staticmethod
+    def _checkpoint_key(checkpoint: Checkpoint) -> CheckpointKey:
+        return CheckpointService.CheckpointKey(
+            view_no=checkpoint.viewNo,
+            pp_seq_no=checkpoint.seqNoEnd,
+            digest=checkpoint.digest
+        )
+
     def process_new_view_accepted(self, msg: NewViewAccepted):
         # 1. update shared data
         cp = msg.checkpoint
         if cp not in self._data.checkpoints:
             self._data.checkpoints.append(cp)
-        self._set_stable_checkpoint(cp.seqNoEnd)
+        self._mark_checkpoint_stable(cp.seqNoEnd)
```

Comment:
Previously `_set_stable_checkpoint` doesn't send `CheckpointStabilized` message, so no GC was called. 
Now it will be sent (and GC will be called) on `NewViewAccepted`. 
It looks like this is fine, just the observation of a changed behavior.